appveyor.yml

```diff
@@ -1,23 +1,44 @@
+#  CSC_KEY_PASSWORD:
+#    secure: as6w3B1D2C5/bOsqBriUeg==
+artifacts:
+  - path: dist
 platform:
   - x64
 init:
   - git config --global core.autocrlf input
+cache:
+  - node_modules
+  - '%LOCALAPPDATA%/Yarn'
 install:
+  - ps: Start-FileDownload 'http://slproweb.com/download/Win64OpenSSL-1_1_0h.exe'
+  - ps: Start-Process 'Win64OpenSSL-1_1_0h.exe' -ArgumentList '/silent /verysilent /sp- /suppressmsgboxes' -Wait
+  - choco install gcloudsdk
+  - refreshenv
   - ps: Install-Product node 9 x64
   - npm i -g --production windows-build-tools
   - npm config set msvs_version 2015 --global
   - set PATH=C:\Ruby22\bin;%PATH%
   - gem install sass compass
   - npm i -g yarn
-  - npm i --no-prune
```

Comment:
this was breaking builds without it - if yarn solves then cool but for me it caused a lot of dependency issues

Reply:
@altninja yeah, with yarn there is not need for it. cheers